src/chat/chatSettings/ChatDetails.tsx

```diff
@@ -41,7 +41,12 @@ const ChatDetails: FunctionComponent<{
       <Text>{chat.name}</Text>
       <Divider style={{ marginVertical: 8 }} />
       <Text category="label">Chat Location:</Text>
+
       <Text
+        accessibilityRole="link"
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
Put a comment above here explaining why you're disabling typescript. Always explain why you disable rules.